content/docs/dvclive/how-it-works.md

```diff
@@ -0,0 +1,59 @@
+# How it Works
+
+DVCLive will store the logged data under the directory (`dir`) passed to
+[`Live()`](/doc/dvclive/api-reference/live). If not provided, `dvclive` will be
+used by default.
+
+The contents of the directory will depend on the methods used:
+
+| Method                                                                    | Writes to                  |
+| ------------------------------------------------------------------------- | -------------------------- |
+| [live.log_metric](/doc/dvclive/api-reference/live/log_metric)             | `dvclive/plots/metrics`    |
+| [live.log_image](/doc/dvclive/api-reference/live/log_image)               | `dvclive/plots/images`     |
+| [live.log_param](/doc/dvclive/api-reference/live/log_param)               | `dvclive/params.yaml`      |
+| [live.log_sklearn_plot](/doc/dvclive/api-reference/live/log_sklearn_plot) | `dvclive/plots/sklearn`    |
+| [live.make_report](/doc/dvclive/api-reference/live/make_report)           | `dvclive/report.{md/html}` |
+| [live.make_summary](/doc/dvclive/api-reference/live/make_summary)         | `dvclive/metrics.json`     |
```

Comment:
Not blocking: somewhere on this page, it could help to mention these are called by `next_step`.